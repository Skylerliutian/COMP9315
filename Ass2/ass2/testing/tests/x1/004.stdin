N A 100
s A 20
s A 25
s A 55
s A 96
O A
S A
O A
u A 66
u A 44
u A 0
O A
U A
O A
F A