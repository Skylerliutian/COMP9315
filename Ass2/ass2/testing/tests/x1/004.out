00000001000000000000000000000000000000000000000010000000000000000000000000000010000100000000000000000000
11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111
11111111111111111111111111111111111110111111111111111111111011111111111111111111111111111111111111111110
00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
